Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2436,6 +2436,23 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
 // TODO: Consider making the return type be generic over `FloatingPoint` types
 // so that `self`'s scalar type can be any `Numeric` type.
 public extension Tensor where Scalar: TensorFlowFloatingPoint {
```

Comment:
`isAxisInRange` and `areAxesInRange` methods are currently defined in `public extension Tensor where Scalar: TensorFlowFloatingPoint`. This is not desirable:
- We don't want to publicly expose these functions. They should have `internal` access.
- We don't want to constrain these functions to `Tensor where Scalar: TensorFlowFloatingPoint`, no `where` clause is necessary.

Please move `isAxisInRange` and `areAxesInRange` into a separate extension (the default access level is `internal`):
```suggestion
extension Tensor {
    ...
}
```